container/typings/LuigiCompoundContainer.svelte.d.ts

```diff
@@ -24,6 +24,12 @@ export default class LuigiCompoundContainer extends HTMLElement {
    */
   deferInit: boolean;
 
+  /**
+   * The parameters to be passed to the compound micro frontend.
```

Comment:
```suggestion
   * The parameters to be passed to the compound micro frontend. Will not be passed to the compound children.
```

Seems like this part is missing here. since we skip it for compound children. 

Also I saw that this sentence "Will not be passed to the compound children." is present on [LuigiContainer Typings : nodeParams ](https://github.com/SAP/luigi/blob/main/container/typings/LuigiContainer.svelte.d.ts#L81) . It should be removed from there and be kept here, since LuigiContainer doesn't deal with compound and that part there is wrong. You can do it in this PR maybe since its related